grakn-graph/src/main/java/ai/grakn/graph/internal/ConceptImpl.java

```diff
@@ -193,6 +181,16 @@ public int compareTo(Concept o) {
         return this.getId().compareTo(o.getId());
     }
 
+    /**
+     * Helper method to cast interfaces into their implementation.
+     *
+     * @param concept The interface to be casted
+     * @return The concept casted to it's implementation
+     */
+    public static ConceptImpl from(Concept concept){
+        return (ConceptImpl) concept;
+    }
```

Comment:
That is a good point actually. It's not idealogical. I't practical. I have gotten these crashes.